### PR TITLE
Don't do a double CI run for PRs!

### PR DESCRIPTION
**Pull request by madwort**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: "jacktrip_pypatcher"
 
-on: [ push, pull_request ]
+on:
+  push:
+    branches:
+      - master
```
Comment (sandreae) [inline .github/workflows/main.yml:6]: Is this right? Or should it be `trunk`?

Comment (madwort) [reply .github/workflows/main.yml:6]: you're right! it should be `trunk`!

Comment (madwort) [reply .github/workflows/main.yml]: updated...

Comment (sandreae) [reply .github/workflows/main.yml]: :+1:

**Review by sandreae — APPROVED** [review]

**Pull request merged**